### PR TITLE
add IPv6 Support

### PR DESCRIPTION
**Pull request by LordSimal**

### CODE REVIEW

File: tests/DownloaderTest.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ public function it_can_download_a_certificate_from_a_host_name_with_strange_char
     public function it_can_download_a_certificate_for_a_host_name_from_an_ip_address()
     {
         $sslCertificate = SslCertificate::download()
-            ->fromIpAddress('138.197.187.74')
+            ->fromIpAddress('164.92.244.169')
```
Comment (LordSimal) [inline tests/DownloaderTest.php:62]: The IP address of your website has changed, thats why this change is here

**Comment by freekmurze** [comment]
Thanks for your work on this.

The tests seem to fail because GitHub actions doesn't seem to reach IPv6 addresses. Do you know how to configure it so it can be reached?

If these no way to achieve this, make it so that the tests is skipped when running on GitHub actions.

**Comment by LordSimal** [comment]
Wow, I didn't expect Github to not be able to reach IPv6 addresses 😂 
But Github seems to not be able to support a IP protocoll which exists for more than 10 years....
https://github.com/actions/virtual-environments/issues/668

Anyway, I will try to find a way to skip that test in Github actions but just so you know that it acutally works:
Here is a screenshot of the tests with IPv6 enabled.
image

**Comment by freekmurze** [comment]
Seems like there are other waiting for IPv6 support on GitHub too: https://github.com/actions/virtual-environments/issues/668

**Comment by freekmurze** [comment]
Perfect, thanks!
